<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="showImgIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>501</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="showImgOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>501</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="maskSize">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>178</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Size(N*N)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convImg">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>278</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Convolution</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maskCoefficients">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>228</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mask Size Coefficients (use space to separate, row-&gt;column)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gaussianImg">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>378</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Marr-Hildreth</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoGImg">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>328</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LoG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="local_enhancement">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>428</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新細明體</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Local_Enhancement 4.1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="local_enhancement_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>480</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新細明體</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Local_Enhancement 4.2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuopen">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionclose"/>
   </widget>
   <addaction name="menuopen"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
